src/applications/vaos/tests/e2e/express-care.cypress.spec.js

```diff
@@ -0,0 +1,29 @@
+import { initExpressCareMocks } from './vaos-cypress-helpers';
+
+describe('Express Care request flow', () => {
+  it('should submit a request successfully', () => {
+    initExpressCareMocks();
+    cy.visit('health-care/schedule-view-va-appointments/appointments/');
+    cy.injectAxe();
+    cy.get('.va-modal-body button').click();
+    cy.wait('@getRequestEligibilityCriteria');
+    cy.axeCheck();
+    cy.findByText('Create an Express Care request').click();
+    cy.findByText('How Express Care works');
+    cy.url().should('include', '/new-express-care-request');
```

Comment:
Might be helpful to break up or group the calls for each page with a comment or line break, so we can see the grouping more clearly.